_automating-configurations/workflow-templates.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+title: Workflow templates
+nav_order: 25
+---
+
+# Workflow templates
+
+OpenSearch provides several workflow templates for some common machine learning (ML) use cases, such as semantic or conversational search. 
+
+You can specify a workflow template when you call the [Create Workflow API]({{site.url}}{{site.baseurl}}/automating-configurations/api/create-workflow/). To provision the workflow, specify `provision=true` as a query parameter. For example, you can configure [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/) by using the `local_neural_sparse_search_bi_encoder` workflow template, as shown in the following request:
+
+```json
+POST /_plugins/_flow_framework/workflow?use_case=local_neural_sparse_search_bi_encoder
+```
+{% include copy-curl.html %}
+
+The workflow created using this template performs the following configuration steps:
+
+- Deploys the default pretrained sparse encoding model (`amazon/neural-sparse/opensearch-neural-sparse-encoding-v1`)
+- Creates an ingest pipeline that contains a `sparse_encoding` processor, which converts the text in a document field to vector embeddings using the deployed model
+- Creates a sample index for sparse search, specifying the default pipeline as the newly created ingest pipeline
```

Comment:
I know I initially said no periods here, but once the fragments become complex/lengthy, as the second two are, then periods are preferred.